api/config/config.go

```diff
@@ -29,8 +31,12 @@ type Config struct {
 	StartGrpcServices   []string `mapstructure:"grpc"`
 	GrpcServerPort      int      `mapstructure:"grpc-port"`
 	GrpcServerInterface string   `mapstructure:"grpc-interface"`
-	StartJSONServer     bool     `mapstructure:"json-server"`
-	JSONServerPort      int      `mapstructure:"json-port"`
+	// GRPC send and receive buffer size
+	GrpcSendBufferSize int `mapstructure:"grpc-send-buffer-size"`
```

Comment:
but it is not exactly about buffer, i think options should be max-send-msg-size, max-recv-msg-size

Reply:
Yeah makes sense. Addressed.